### PR TITLE
Add API support for editing any tweak

### PR DESCRIPTION
**Pull request by henriquegemignani**

### CODE REVIEW

File: src/open_prime_rando/echoes/schema.json [hunk 1]
```diff
@@ -97,6 +97,43 @@
...
+                    "TweakPlayerControls",
+                    "TweakPlayerGunMuli",
+                    "TweakPlayerGun",
+                    "TweakCameraBob",
+                    "TweakPlayer"
+                ]
+            },
+            "additionalProperties": {
+                "type": "object",
+                "description": "Mapping of full property path to new value. For nested properties, include parent property names split with .",
```
Comment (duncathan) [inline src/open_prime_rando/echoes/schema.json:126]: what about unknowns/unnamed properties? maybe include an example of one

**Review by duncathan — CHANGES_REQUESTED** [review]

File: src/open_prime_rando/echoes/schema.json (hunk 1 above)
Comment (henriquegemignani) [reply src/open_prime_rando/echoes/schema.json:126]: Currently needs to use the `unknown_0x2e997e0b` style, but it does remind me that if we add a proper name for these in RDS they'll break. I guess that's fine?

Comment (duncathan) [reply src/open_prime_rando/echoes/schema.json:126]: yeah that's an existing issue elsewhere anyway. an example using that style should be plenty then

**Comment by Miepee** [comment]
Does this fix #42 ?

**Comment by henriquegemignani** [comment]
It can be used to do it

**Review by duncathan — APPROVED** [review]

**Pull request merged**